mkdocs.yml

```
@@ -26,6 +26,7 @@ theme:
     - toc.integrate
 
 repo_url: https://github.com/Azure/iotedge-lorawan-starterkit
+edit_uri: tree/docs/main/docs
```

Think it should read as follows?

```suggestion
edit_uri: edit/docs/main/docs/
```